### PR TITLE
#59:

### PR DESCRIPTION
**Pull request by jasonmerry**
updated 7zip to latest version

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "electron-devtools-installer",
-  "version": "2.2.0",
+  "version": "2.2.1",
```
Comment (MarshallOfSound) [inline package.json]: Please don't bump the version of this package, I'll do that when I want to release a new version

File: package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
     "spec-xunit-file": "0.0.1-3"
   },
   "dependencies": {
-    "7zip": "0.0.6",
+    "7zip": "^0.1.1",
```
Comment (MarshallOfSound) [inline package.json]: Please version lock this dependency

Comment (jasonmerry) [inline package.json]: Sure, doing now :)

Comment (jasonmerry) [inline package.json]: done

File: package.json (hunk 1 above)
Comment (jasonmerry) [inline package.json]: sorry :(

**Comment by jasonmerry** [comment]
So it appears that although the git repo has been updated, the npm package has not. :(

**Comment by MarshallOfSound** [comment]
I'm still confused as to why we need this change, `electron-devtools-installer` works perfectly for me on latest master on windows.

**Comment by MarshallOfSound** [comment]
To prove this fact 😆 I just set up AppVeyor CI for windows tests, and as you can see current master is passing just fine.

**Comment by jasonmerry** [comment]
without updating the `7zip` to `win-7zip` webpack is spitting out this error:
[https://github.com/MarshallOfSound/electron-devtools-installer/pull/53](url)

I don't doubt it is building fine for you, but it still isn't working for me without the update to `win-7zip`.
- using `electron: ^1.6.2`
- using `webpack: ^2.2.1`

after importing as per README I am adding `installExtension(REACT_DEVELOPER_TOOLS);` within my function to create my electron window.

If you know how I can allow my project to change
`var bin = require('./package').bin` to access `var bin = require('./package.json').bin` automatically from any dependency I'd be happy to see if that works.

**Comment by MarshallOfSound** [comment]
@jasonmerry You probably just need to add `json` to your list of extensions for webpack to resolve by default

**Comment by jasonmerry** [comment]
got that already
```
{
    test: /\.json$/,
    use: 'json-loader'
},
```

**Comment by MarshallOfSound** [comment]
@jasonmerry You should be looking at your `resolve.extensions` config

https://webpack.js.org/configuration/resolve/#resolve-extensions

**Comment by jasonmerry** [comment]
Adding ".json" to my resolves worked a treat :)

Thanks

So, do I cancel the pull request or do you?

**Comment by MarshallOfSound** [comment]
For anyon finding this in the future, this isn't be saying I won't update dependencies.  This is purely a risk management tactic.  If the issue can be fixed in user config in a third party module I don't want to take on board the risk of updating dependencies 👍

**Comment by timsuchanek** [comment]
It's also not working for me, same issue as @jasonmerry . It would be great if this PR could be merged if it doesn't break anything. I will have to use the custom fork of @jasonmerry now to make it run

**Comment by MarshallOfSound** [comment]
@timsuchanek Have you tried applying the solution that worked for @jasonmerry

**Comment by jasonmerry** [comment]
@timsuchanek I deleted my custom fork because the fix, adding `.json` to the webpack config.

Follow the link previously in this thread or for a quick example, add this somewhere in your config:
```
resolve: {
    extensions: ['.json']
  }
```
If I had known it was that easy in the first place I wouldn't have created the fork.

**Comment by BeeeQueue** [comment]
I'm having this issue as well, and `.json` is already in my extensions.

```js
extensions: [
  '.js',
  '.jsx',
  '.vue',
  '.json',
  '.ts',
  '.tsx'
],
```

**Pull request closed without merging**